modules/ao_limit/init.sqf

```diff
@@ -45,7 +45,7 @@ if (!isDedicated) then {
 					} forEach _markers;
 					
 					if (_outside) then {
-						if (!(_allowedOutside) && (_vehicle call FNC_Alive) && local _vehicle) then {
+						if (!(_allowedOutside) && (_vehicle call FUNC(isAlive)) && local _vehicle) then {
```

Comment:
Wasn't this function changed from `isAlive` to `alive` much further up the list of file changes?

Reply:
No, it has always been FNC_alive in olsen

Reply:
Fair enough.